src/screens/Proposals.js

```diff
@@ -118,6 +119,13 @@ const Proposals = React.memo(
       [history]
     )
 
+    let current_entries_per_page = ENTRIES_PER_PAGE[0]
+    let selected_entries_per_page = 1
+    const updateProposalsPerPage = useCallback((selected, value) => {
+      current_entries_per_page = ENTRIES_PER_PAGE[selected]
```

Comment:
I think that using a variable to store state is probably not the best option in React, considering it's lifecycle. Maybe using `useState()` to keep track/change this variable would be better!